Dockerfile

```
@@ -5,6 +5,10 @@ WORKDIR /elk
 
 FROM base AS builder
 
+# Make two NUXT_PUBLIC vars changeable during builds
+ARG NUXT_PUBLIC_SINGLE_INSTANCE
```

```suggestion
ARG NUXT_PUBLIC_SINGLE_INSTANCE
ENV NUXT_PUBLIC_SINGLE_INSTANCE=$NUXT_PUBLIC_SINGLE_INSTANCE
```

My understanding is an ARG is usable in a Dockerfile, but to make it usable in things like `npm build` we need to explicitly set it to an ENV variable